static/scss/pages/faq.scss

```diff
@@ -0,0 +1,53 @@
+.faqs-wrapper {
+    max-width: 1100px;
+}
+
+.faqs {
+    column-width: $content-sm;
+    column-gap: $spacing-2xl;
+}
+
+.faq {
+    margin-bottom: 3em;
+    break-inside: avoid;
+}
+
+.faqs-headline {
+    margin: $spacing-2xl 0 $spacing-xl;
+    font-size: 2em;
+}
+
+.faqs-headline,
+.faq-headline {
+    color: $color-white;
+}
+
+.faq-headline {
+    font-size: 1.25em;
+    line-height: 1.3;
+    margin-top: 0;
+    margin-bottom: .5em;
+}
+
+.faq-answer {
+    color: $color-grey-10;
+    font-size: 1em;
```

Comment:
Use the [typography sizing](https://protocol.mozilla.org/fundamentals/typography.html#code) from Protocol to set the size.